cmd/accountsmgr-server/main.go

```diff
@@ -0,0 +1,46 @@
+package main
+
+import (
+	AccountsMgr "accountsmgr"
+	"accountsmgr/api/handlers"
+	"accountsmgr/gen/restapi"
+	"accountsmgr/gen/restapi/operations"
+	"flag"
+	"log"
+
+	"github.com/go-openapi/loads"
+	//_ "accountsmgr/db/inmemory"
+	_ "accountsmgr/db/mongo"
+)
+
+func main() {
+
+	var portFlag = flag.Int("port", 4000, "Port to run this service on")
+
+	// load embedded swagger file
+	swaggerSpec, err := loads.Analyzed(restapi.SwaggerJSON, "")
+	if err != nil {
+		log.Fatalln(err)
+	}
+
+	// create new service API
+	api := operations.NewAccountsmgrAPI(swaggerSpec)
+
+	v := AccountsMgr.NewRunTime("Gagan")
```

Comment:
Gagan????
is this your application name???

Reply:
Done. Kindly Check.